playbooks/group_vars/all/global.yml

```diff
@@ -1,108 +1,37 @@
 ---
```

Comment:
Versions of minishift should be pulled from minishift and openshift directly since you can determine them easily without going through the github UI

Reply:
I had many problems with a 'rate limit' of API of GitHub [1] [2]. It's not related to UI of GitHub. I don't have this issue outside of the offices but in the places that have many guys downloading the same release have this limitation. We can offer to contra-env-user users a place like minishift.iso to reduce this problem while it is not fixed by minishift and openshift or we can require to users should create a Personal Token in GitHub to start use based in this limitation. What're the options do you recommend? 
[1] https://github.com/minishift/minishift/issues/1920
[2] https://github.com/openshift/origin/issues/18961